api/v1beta1/cryostat_types.go

```diff
@@ -55,11 +55,6 @@ type CryostatSpec struct {
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	ReportOptions *ReportConfiguration `json:"reportOptions,omitempty"`
-	// The maximum number of WebSocket client connections allowed (minimum 1, default unlimited).
-	// +optional
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Max WebSocket Connections",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:number"}
-	// +kubebuilder:validation:Minimum=1
-	MaxWsConnections int32 `json:"maxWsConnections,omitempty"`
```

Comment:
I guess maybe this should be kept in the `v1beta1` spec so that it doesn't break API compatibility, but the value would actually just be ignored by the implementation and not handled on upgrades to `v1beta2`.